spec/requests/fitness_logs_spec.rb

```diff
@@ -15,5 +15,26 @@
      expect(response).to have_http_status(200)
      expect(fitnesslogs.length).to eq 1
    end
+end
+   describe "POST /create" do
+    it 'creates an fitness log' do
+      fitness_log_params = {
+        fitnesslog: {
+          date: DateTime.now,
+          workout_id: 874,
+          workout_name: 'Forward Lunge',
+          happy: 5,
+          sad: 5,
+          energetic: 7,
+          lethargic: 2,
+          entry: 'crushed it!',
+          user_id: user.id,
+        }
+    }
+      post '/fitness_logs', params: fitness_log_params
+      new_fitness_log = FitnessLog.first
+      expect(response).to have_http_status(200)
+      expect(new_fitness_log.sad).to eq 5
```

Comment:
You should probably test each column in fitness log.

Reply:
Complete